src/CommentsList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+
+const CommentsList = ({ comments }) => (
+  <div className="post__comments comments">
+    <p>Comments:</p>
+    <ul className="comments__list">
+      {comments.map(comment => <Comment comment={comment} />)}
```

Comment:
4(четыре!) слова сomment подряд
```suggestion
      {comments.map(singleComment => <Comment comment={singleComment} />)}
```